File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    int t=0;
    for(int i=0;i<s.length();i++) {
        if(s[i]==' ') {
            t=0;
        } else {
            s[i]=(t++%2==0)?toupper(s[i]):tolower(s[i]);
        }
    }
    string r=s;
    return r;
}

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.java
class Solution {
    public String solution(String s) {
        String rlt = "",tmp[]=s.split("");
        int cnt = 0;
        for(String st:tmp) {
            cnt=st.contains(" ")?0:cnt+1;
            rlt+=cnt%2==0?st.toLowerCase():st.toUpperCase();
        }
        return rlt;
    }
}